features/step_definitions/inspection_steps.rb

```diff
@@ -2,6 +2,10 @@
   expect(page).to have_xpath("//fieldset/legend[contains(., '#{legend}')]", visible: false)
 end
 
+Then(/^I should see a title called "(.*?)"$/) do |title|
```

Comment:
Extreme nitpick: Could we use "heading" here?  Title sounds a little like we mean the browser ``<title>`` element.